tutorials/fmc/fmc.c

```diff
@@ -0,0 +1,59 @@
+#include "fmc_config.h"
+#include "libs/adc/api.h"
+#include "libs/timer/api.h"
+#include "tutorials/fmc/can_api.h"
+
+#include <avr/interrupt.h>
+#include <avr/io.h>
+
+volatile bool timer_50hz = true;
+uint16_t adc_value = 0;
+bool high_voltage = false;
+uint8_t high_voltage_counter = 0;
+uint8_t buffer_counter = 0;
+
+void timer0_callback(void) {
+    timer_50hz = true;
+}
+
+void change_duty_cycle(uint8_t torque_request_value) {
+    float duty_cycle = (float)torque_request_value / 255.0;
+    OCR1B = (uint16_t)duty_cycle * 80.0;
```

Comment:
Where did 80 come from? Anytime you have numbers appear in your code, you should move them to `#define` statements and leave a comment wherever you define them about their purpose. Just having `80.0` in there is called a "magic number" because it is magic and it's hard to tell what it means.